TESTING.rst

```diff
@@ -61,20 +61,51 @@ Running Unit Tests from PyCharm IDE
 To run unit tests from the PyCharm IDE, create the `local virtualenv <LOCAL_VIRTUALENV.rst>`_,
 select it as the default project's environment, then configure your test runner:
 
-.. image:: images/configure_test_runner.png
+.. image:: images/pycharm/configure_test_runner.png
     :align: center
     :alt: Configuring test runner
 
 and run unit tests as follows:
 
-.. image:: images/running_unittests.png
+.. image:: images/pycharm/running_unittests.png
     :align: center
     :alt: Running unit tests
 
 **NOTE:** You can run the unit tests in the standalone local virtualenv
 (with no Breeze installed) if they do not have dependencies such as
 Postgres/MySQL/Hadoop/etc.
 
+Running Unit Tests from PyCharm IDE using Breeze
+------------------------------------------------
+
+Ideally, all unit tests should be run using the standardized Breeze environment.  While not
+as convenient as the one-click "play button" in PyCharm, the IDE can be configured to do
+this in two clicks.
+
+1. Add Breeze as an "External Tool"
+  a. File > Settings > Tools > External Tools
+  b. Click the little plus symbol to open the "Create Tool" popup and fill it out:
```

Comment:
Some macOS specific stuff. 
On macOS (and only there) user should navigate to `PyCharm -> Preferences` instead of `File > Settings`

Reply:
Apple: "Think Different" 🙄

Reply:
Rephrased to "From the settings menu,"  Hopefully we can safely assume that a user can find where settings menu is in their OS of choice, I guess.